pysgrid/sgrid.py

```diff
@@ -514,14 +525,15 @@ def save_as_netcdf(self, filepath):
             # populate variables with data
             grid_time[:] = self.grid_times[:]
             grid_center_lon[:] = self.centers[..., 0]
-            grid_center_lat[:] = self.centers[..., 1] 
-            grid_node_lon[:] = self.nodes[..., 0]
-            grid_node_lat[:] = self.nodes[..., 1]
+            grid_center_lat[:] = self.centers[..., 1]
+            if self.nodes is not None: 
+                grid_node_lon[:] = self.nodes[..., 0]
+                grid_node_lat[:] = self.nodes[..., 1]
```

Comment:
>I do not see placing it on a branch as " cowboy SGrid3D's removal", in fact that is the best Open Source way to discuss an idea. Not in master...

I disagree. I think branches tend to obscure the work that is being done and increases risk to a project as they encourage larger merges. Other contributors can easily see the direction the code is going, rather than have it obfuscated in a branch whose relationship to master may be nebulous. I think we can agree to disagree on this topic however.